modules/rollup-plugin-workbox-inject/index.js

```diff
@@ -13,14 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const escapeRegexp = require('workbox-build/src/lib/escape-regexp');
+
+const chalk = require('chalk');
+const defaults = require('workbox-build/src/options/defaults');
 const getFileManifestEntries = require('workbox-build/src/lib/get-file-manifest-entries');
-const injectManifestSchema = require('workbox-build/src/options/schema/inject-manifest');
+const getManifestSchema = require('workbox-build/src/options/schema/get-manifest');
```

Comment:
Now that `swSrc` and `swDest` aren't specified anymore, we can switch to the actual `getManifest` schema.